cpgserver/build.sbt

```diff
@@ -1,4 +1,4 @@
-val ScalatraVersion = "2.6.5"
+val ScalatraVersion = "2.7.0-RC1"
```

Comment:
that seems unnecessarily risky?
If this is because of the Scala upgrade maybe we should wait until things are upgraded fully?

Reply:
scalatra doesn't do much in our setup, it's only used in cpg-server, and only does one minimal thing. if all tests pass, I'd rather not wait

Reply:
ok